class.c

```diff
@@ -771,7 +771,8 @@ Perl_class_seal_stash(pTHX_ HV *stash)
             }
 
             UNOP_AUX_item *aux;
-            Newx(aux, 2, UNOP_AUX_item);
+            aux = (UNOP_AUX_item*)PerlMemShared_malloc(
```

Comment:
Most minor of nitpicks: Style-wise I tend to put a space between`UNOP_AUX_item` and `*`